cli/options/src/lib.rs

```diff
@@ -132,8 +132,10 @@ pub enum Backend {
     Fstar(FStarOptions),
     /// Use the Coq backend
     Coq,
-    /// Use the EasyCrypt backend
+    /// Use the EasyCrypt backend (warning: work in progress!)
     Easycrypt,
+    /// Use the ProVerif backend (warning: work in progress!)
```

Comment:
I'll take care of putting in a warning in the CLI output when `into pro-verif` is invoked.

Reply:
Ok, great!